app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -140,7 +133,7 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
           setInProgress(false);
         });
     },
-    [locationData, inProgress],
+    [locationResults, inProgress],
   );
 
   const handleFoundAddress = useCallback((address) => {
```

Comment:
When there are no results in the search, I see an error in the console stemming from this callback assuming an `address` is present:

>Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'street_address')

Not sure if we want to fix that as part of this work, or if we have a separate ticket we can track that in.